### ISSUE TITLE
activity page 500 error

### ISSUE THREAD

**Issue by chian-007** [ckan/ckan#8355]
## Describe the bug
the activity page got the 500 error:
![image](https://github.com/user-attachments/assets/552eb70f-8460-4cd8-bcec-64426821171d)

  File "/usr/lib/ckan/venv/src/ckan/ckanext/activity/templates/snippets/activities/deleted_organization.html", line 9, in top-level template code
  File "/usr/lib/ckan/venv/lib/python3.8/site-packages/jinja2/runtime.py", line 777, in _invoke
    rv = self._func(*arguments)
  File "/usr/lib/ckan/venv/src/ckan/ckanext/activity/templates/snippets/stream.html", line 17, in template
    <a href="{{ h.url_for(group_type ~ '.read', id=activity.object_id) }}">
  File "/usr/lib/ckan/venv/src/ckan/ckan/lib/helpers.py", line 385, in url_for
    my_url = _url_for_flask(*args, **kw)
  File "/usr/lib/ckan/venv/src/ckan/ckan/lib/helpers.py", line 443, in _url_for_flask
    my_url = _flask_default_url_for(*args, **kw)
  File "/usr/lib/ckan/venv/lib/python3.8/site-packages/flask/helpers.py", line 338, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/usr/lib/ckan/venv/lib/python3.8/site-packages/flask/helpers.py", line 325, in url_for
    rv = url_adapter.build(
  File "/usr/lib/ckan/venv/lib/python3.8/site-packages/werkzeug/routing.py", line 2315, in build
    raise BuildError(endpoint, values, method, self)

### Steps to reproduce

1 .login the user without any activity with any the following:
create, edit or delete organization.
2. go to the activity page and it will show all the activities

3. then do one of the above activities
4. go to the activity page and it will show the wrong page
5. delete the activities with organization in the table: activity under the ckan database
6.  go to the activity page and it will show all the activities normally

### Expected behavior

there is no error to show all the activities.

**Comment by amercader** [comment]
@chian-007 are there further details below or above the error lines you pasted? This error means that werkzeug can't generate a URL for an endpoint, but we don't know the endpoint is trying to use.

Are you using a custom schema for organizations (.i.e. `scheming.organization_schemas` in your ini file)? I can't reproduce this in a standard CKAN

![Screenshot 2024-07-23 at 15-43-50 Manage - amercader - Users - CKAN](https://github.com/user-attachments/assets/ef249366-bc20-4a91-b612-f9f18e96c264)

**Comment by chian-007** [comment]
yes, we do use the scheming_organizations. I tested it without scheming_organizations and it works. Therefore, is it possible to make it work with the scheming_organizations in the new version of ckan. Thanks
